packages/frontend/src/components/staking/components/Validators.js

```diff
@@ -38,12 +50,14 @@ export default function Validators({ validators, stakeFromAccount }) {
                 </div>
             )}
             <ListWrapper>
-                {validators.filter(v => v.accountId.includes(validator)).map((validator, i) => 
-                    <ValidatorBox
-                        key={i}
-                        validator={validator}
-                    />
-                )}
+                {
+                    filterValidators.map((validator, i) =>
+                        <ValidatorBox
+                            key={i}
```

Comment:
Not a blocker since this was already wrong, but the `key` property should be the validator account _name_ instead of an index into the array of validators (key as array index is a react antipattern) -- we can fix this in the next PR